modules/dhcp_common/dhcp.rb

```diff
@@ -0,0 +1,26 @@
+module Proxy::DHCP
+  Standard = {
+      :hostname              => {:code => 12, :kind => "String"    }, # The host's name
+      :PXEClient             => {:code => 60, :kind => "String"    }, # Needs to be empty for foreman to function
+      :nextServer            => {:code => 66, :kind => "String"    }, # From where we download the pxeboot image via TFTP
+      :filename              => {:code => 67, :kind => "String"    }  # The pxeboot image
+  }
+  SUNW     = {
+      :root_server_ip        => {:code => 2,  :kind => "IPAddress" }, # 192.168.216.241
+      :root_server_hostname  => {:code => 3,  :kind => "String"    }, # mediahost
+      :root_path_name        => {:code => 4,  :kind => "String"    }, # /vol/solgi_5.10/sol10_hw0910/Solaris_10/Tools/Boot
+      :install_server_ip     => {:code => 10, :kind => "IPAddress" }, # 192.168.216.241
+      :install_server_name   => {:code => 11, :kind => "String"    }, # mediahost
+      :install_path          => {:code => 12, :kind => "String"    }, # /vol/solgi_5.10/sol10_hw0910
+      :sysid_server_path     => {:code => 13, :kind => "String"    }, # 192.168.216.241:/vol/jumpstart/sysidcfg/sysidcfg_primary
+      :jumpstart_server_path => {:code => 14, :kind => "String"    }  # 192.168.216.241:/vol/jumpstart
+  }
+  class Error < RuntimeError; end
```

Comment:
Nitpick, this layout's a bit different to https://github.com/theforeman/smart-proxy/pull/329/files#diff-8896d0ca8444f24cff56fc1d0a1af45d, it might be best to be consistent - either a separate errors file or common file like this?